resources/ory/charts/hydra/charts/hydra-maester/crds/crd-oauth2clients.yaml

```diff
@@ -0,0 +1,199 @@
+
```

Comment:
We keep CRDs in kyma/installation/resources/crds/ory, so I think this is not needed